src/libretro/message/error.cpp

```diff
@@ -36,7 +36,7 @@ constexpr pntr_color TEXT_COLOR_BOTTOM = {.rgba = {.b = 0x98, .g = 0xE5, .r = 0x
 
 static constexpr const char* const ERROR_TITLE = "Oh no! melonDS DS couldn't start...";
 static constexpr const char* const SOLUTION_TITLE = "Here's what you can do:";
-static constexpr const char* const THANK_YOU = "Thank you for using melonDS DS!";
+static constexpr const char* const THANK_YOU = "Thank you for using melonDS DS! Θ.Θ";
```

Comment:
Got it!

![Screenshot from 2024-04-20 04-35-22](https://github.com/JesseTG/melonds-ds/assets/25086/fc417f39-6140-4159-826c-61d0f4a73574)